flang/lib/Optimizer/CodeGen/CMakeLists.txt

```diff
@@ -11,6 +28,8 @@ add_flang_library(FIRCodeGen
   TargetRewrite.cpp
   TypeConverter.cpp
 
+  PARTIAL_SOURCES_INTENDED
```

Comment:
Thank you for the review, Tom!

I will get rid of these `PARTIAL_SOURCES_INTENDED` in this PR in the next commit.  It was just a workaround to upload the rest of the functional changes.